### PR TITLE
Feature/profile page

### LINKED ISSUE #23: Create a Profile page
Create a profile page, either using activity or fragment, displaying general information of the user. It may be opened through the navigation drawer.

### PR DESCRIPTION
**Pull request by vishal2376**
## Description

- Create a basic profile page
- Move to `ProfileActivity` by clicking the profile button (Navigation drawer)

## Issue Resolved

- Fix issue #23 

## Screenshots 

image

## Checklist

Please make sure to review the following before submitting your PR: 

- [x] I have read the [contribution guidelines](https://github.com/PEC-CSS/MovieDroid/blob/main/CONTRIBUTING.md).
- [x] I have read the code of conduct.
- [x] I have reviewed my submission in detail.

### CODE REVIEW

File: app/src/main/java/com/pec_acm/moviedroid/ProfileActivity.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+class ProfileActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_profile)
+    }
+}
```
Comment (ken1000minus7) [inline app/src/main/java/com/pec_acm/moviedroid/ProfileActivity.kt]: Please add the functionality to display the name, image and email of the user in the corresponding views. Make sure to use view binding for accessing the views.

File: build.gradle [hunk 2]
```diff
@@ -25,7 +25,7 @@ buildscript {
 plugins {
     id 'com.android.application' version '7.1.0' apply false
     id 'com.android.library' version '7.1.0' apply false
-    id 'org.jetbrains.kotlin.android' version '1.6.21' apply false
+    id 'org.jetbrains.kotlin.android' version '1.7.10' apply false
```
Comment (ken1000minus7) [inline build.gradle]: Please remove this change, it isn't required

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]
Great work @vishal2376 😄 ! However, there are a few changes to be made. After completing them, your pr will be ready to be merged 😇

File: build.gradle (hunk 2 above)
Comment (vishal2376) [reply build.gradle]: ok

File: app/src/main/java/com/pec_acm/moviedroid/ProfileActivity.kt (hunk 1 above)
Comment (vishal2376) [reply app/src/main/java/com/pec_acm/moviedroid/ProfileActivity.kt]: From which activity i can get this info?

Comment (ken1000minus7) [reply app/src/main/java/com/pec_acm/moviedroid/ProfileActivity.kt]: You can get the current user through this
```kt
val user = FirebaseAuth.getInstance().currentUser
```
and then access the `displayName` , `photoUrl` and `email`  from this `user` object 😇

An example of this:
https://github.com/PEC-CSS/MovieDroid/blob/b693a46cc030007abeb4bec2fd0b383b34902355/app/src/main/java/com/pec_acm/moviedroid/MainActivity.kt#L65-L71

**Comment by vishal2376** [comment]
done

**Review by ken1000minus7 — APPROVED** [review]

**Comment by ken1000minus7** [comment]
Lgtm, great work 🎉

**Pull request merged**